portfolio/src/main/webapp/script.js

```diff
@@ -12,14 +12,27 @@
 // See the License for the specific language governing permissions and
 // limitations under the License.
 
-/**
- * Adds a random greeting to the page.
+/** Retrieves data from the /data page and displays it. */
+function getComments() {
+  fetch('/data').then(response => response.json()).then((json) => {
+    json.forEach(commentText => createPElement(commentText));
+  });
+}
+
+/** Creates paragraph elements and sets the text to comments 
+ *   pulled from \data page. 
```

Comment:
If there are multiple lines in a javadoc it's practice to keep the comment-markers separate (without text).
Ie.
/**
 * some text here
 * some more tex
 */

See https://google.github.io/styleguide/javaguide.html#s7.1.1-javadoc-multi-line

Reply:
Done.